scene/main/http_request.cpp

```diff
@@ -207,6 +207,16 @@ void HTTPRequest::cancel_request() {
 	requesting = false;
 }
 
+Error HTTPRequest::_get_redirect_headers(Vector<String> *r_headers) {
+	for (const String &E : headers) {
+		// We strip content headers when changing a redirect to GET.
+		if (!E.begins_with("Content-Type: ") && !E.begins_with("Content-Length: ") && !E.begins_with("Content-Location: ") && !E.begins_with("Content-Encoding: ") && !E.begins_with("Transfer-Encoding: ") && !E.begins_with("Connection: ") && !E.begins_with("Authorization: ")) {
```

Comment:
Headers are case insensitive according to the standard, and the space after the ":" is not required (see [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers)).

```suggestion
	for (const String &E : headers) {
		const String h = E.to_lower();
		// We strip content headers when changing a redirect to GET.
		if (!h.begins_with("content-type:") && !h.begins_with("content-length:") && !h.begins_with("content-location:") && !h.begins_with("content-encoding:") && !h.begins_with("transfer-encoding:") && !h.begins_with("connection:") && !h.begins_with("authorization:")) {
```